docs/deployment.md

```diff
@@ -0,0 +1,99 @@
+# Deployment
+
+1. Create a Google Cloud Project if you do not have one. Enable the Cloud Build and Container Registry APIs.
+
+2. Set the project and compute zone.
+
+```sh
+gcloud config set project $PROJECT_ID
+gcloud config set compute/zone $COMPUTE_ZONE
+```
+
+3. Create a GKE cluster with [Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) enabled. Replace `$CLUSTER_NAME` and `$PROJECT_ID` with the name of the new cluster and the project ID respectively.
```

Comment:
Can you  group these individual instructions into higher level objectives . For example Point 4, 5 can be grouped as 4. Setting up GCSFuse for the project 
      i) Create serivce account ...
      ii) Add policy binding ...
etc.